lib/finance/banding_tracker/participant_per_declaration_type_per_band_aggregator.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
as an aside i don't think we should be adding any new code to `/lib`. If we just namespace correctly in `app/`, zeitwerk and rails should load everything correctly and we don't need to require things manually

Reply:
it is a standalone library, requiring file is fine, it's actually a bit more explicit, at least you know where the code came from. Helps defining boundaries in the code. 
I think we should add more code to `lib` so that we start thinking about problems as domain and respect the boundaries.